### ISSUE TITLE
[BUG] Crash when getting a badge (?)

### ISSUE THREAD

**Issue by Arinono** [pokeclicker/pokeclicker#2265]
**Info**
 - OS: macos 12.4 (arm64)
 - Browser: Brave (Chromium 101.0.4951.67)
 - Version: `v0.9.4`

**Description:**
The game stopped/crashed when I finished the duel vs Winona for the 1st time.
Some things where still working (my Lucky Incense ran out properly). I'm just stuck on the last pokemon w/ a depleted health bar. The combat does not end, I don't get the badge.

**Steps to Reproduce:**
1. Go to a/any Gym for the 1st time
2. Win the fight
3. The game is locked (some parts of it)

**Expected Behavior:**
I get my badge and can move on

**Screenshots:**
![Screenshot 2022-05-26 at 10 28 16](https://user-images.githubusercontent.com/10957531/170451380-d122e0b0-101c-4c0d-86db-1d58b4e2f6c3.png)

**Additional Information:**
A refresh unlocked me (and also got me the pop up to chose my starter, it's unrelated, but I reached Hoenn before the `0.9.4` update)

Here is the stack trace in the screenshot:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'multipliers')
    at Party.calculateOnePokemonAttack (script.min.js?v=0.9.4:37502:23)
    at Party.calculatePokemonAttack (script.min.js?v=0.9.4:37471:28)
    at BadgeCase.gainBadge (BadgeCase.ts:30:264)
    at Gym.firstWinReward (script.min.js?v=0.9.4:35359:28)
    at Function.gymWon (script.min.js?v=0.9.4:36346:21)
    at Function.defeatPokemon (script.min.js?v=0.9.4:35426:23)
    at Function.clickAttack (script.min.js?v=0.9.4:21781:18)
    at Function.clickAttack (script.min.js?v=0.9.4:35414:19)
    at Game.eval (eval at parseBindingsString (knockout-latest.js:74:172), <anonymous>:3:86)
    at HTMLDivElement.<anonymous> (knockout-latest.js:100:264)
```

**Comment by PixLSteam** [comment]
Fixed via #2258

**Comment by Arinono** [comment]
Ha, my bad, I looked at the issues to see if something was already open, but not the PRs 👌
